app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java

```diff
@@ -1363,6 +1363,21 @@ public boolean isOverrideEnvPathEnabled() {
         return SettingsStore.getInstance(this).isEnvironmentOverrideEnabled();
     }
 
+    @Keep
+    @SuppressWarnings("unused")
+    public void checkTogglePassthrough() {
+        switch (DeviceType.getType()) {
+            case DeviceType.LenovoA3:
```

Comment:
> YEah, I think it's still worth pursuing. We have request from some devices to change the default so Wolvic starts with passthrough enabled.

@javifernandez We can add more device types here to make passthrough the default mode at startup.